flow/layers/backdrop_filter_layer.cc

```diff
@@ -43,6 +43,10 @@ void BackdropFilterLayer::Preroll(PrerollContext* context,
                                   const SkMatrix& matrix) {
   Layer::AutoPrerollSaveLayerState save =
       Layer::AutoPrerollSaveLayerState::Create(context, true, bool(filter_));
+  if (context->view_embedder != nullptr) {
+    if (context->view_embedder->GetFoundPlatformViews())
```

Comment:
Do we need this? If there are no platform views, `PushFilterToVisitedPlatformViews` is going to be a no-op anyway because the list is empty.

Reply:
It can be removed. I wasn't sure if it was better to check if we found a platform view first or to just run the method every time even if its no-op

Reply:
Yeah, it's ok to run the no-op every time. the check is redundant.

Reply:
done